recipes/pyem/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "pyem" %}
+{% set version = "0.62" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/asarnow/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a3e97c2ca83229617b04babf8912ef11e9258b225b2c3e559d654886bdc32d16
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win or py<39 or py>311]
```

```suggestion
  noarch: python
```

Why only Python 3.10 and Unix? We can express that with requirements metadata (see below), but I want to make sure I understand why this is the case first.